README.md

```diff
@@ -50,17 +50,16 @@ module.exports = {
 + `sourceMap`: Default: uses [webpackConfig.devtool](https://webpack.js.org/configuration/devtool/). Set this to override that.
 + `parserOpts`: Configure babel with special parser options.
 + `babel`: Pass in a custom babel-core instead. `require("babel-core")`
-+ `babili`: Pass in a custom babili preset instead - `require("babel-preset-babili")`.
-``
++ `minifyPreset`: Pass in a custom minify preset instead - `require("babel-preset-minify")`.
```

Comment:
minify or minifier sends wrong information. For example, when `require("babel-minify")` is passed. IMO, `minifyPreset` says that it's `babel-preset-minify` and is clear.